package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@m0dch3n/vue-cli-plugin-cordova",
-  "version": "0.0.1",
+  "name": "vue-cli-plugin-cordova",
```

Comment:
Notice this diff, it's not directly related to this PR, I can remove it if you prefer.

Reply:
yes, please remove it, the name was changed here https://github.com/m0dch3n/vue-cli-plugin-cordova/commit/f29b68450f5dcf4e71d6660dd8ef3b2f47cb9a8a because of https://github.com/m0dch3n/vue-cli-plugin-cordova/issues/2